### ISSUE TITLE
Replace currently-playing icon with low-CPU alternative

### ISSUE THREAD

**Issue by wmertens** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#3100]
**OS:** macOS 10.13.4

**GPMDP Version:** 4.5.0

**Issue Descriptions:**

The "currently playing" icon that is on a song/album/station when it is being played uses a lot of CPU. It would be great if it could be replaced with something more static.

**Steps to Reproduce:**

* open activity monitor, search for "music"
* start playing song, make sure animated playing icon is visible. Note the CPU usage.
* scroll up the window or navigate away so the icon is not visible. Note the CPU usage.
* now perform these last two steps again, but also minimize and watch CPU usage.

This is a different issue from #2766 - that is also happening, but this issue is separate and also affects CPU usage even when minimized.

**Comment by MarshallOfSound** [comment]
Cursory tests indicate having 1 playing indicator visisble with the window not-minized results in 0.8~1.2% CPU on my machine (it's a pretty decent machine 😆 )

Would 100% accept a PR to change this icon somehow to save CPU cycles 🚔

**Comment by wmertens** [comment]
yeah on my MBP2015 it eats at least 5%… is there a way to bring up the
devtools in the app?

**Comment by MarshallOfSound** [comment]
@wmertens Command + Shift + G

Then type in DEV_MODE and the app should relaunch with the ability to open dev tools 😀

**Comment by wmertens** [comment]
sneaky :)
